front/app/components/UI/QuillEditor/index.tsx

```diff
@@ -282,6 +282,8 @@ class ImageFormat extends BaseImageFormat {
     if (attributes.indexOf(name) > -1) {
       if (value) {
         this.domNode.setAttribute(name, value);
+      } else if (name === 'alt') {
```

Comment:
If alt text is not added to an image, we are adding the attribute anyway with an empty string so that screen readers skip it, instead of reading the file name.